cellprofiler/modules/namesandtypes.py

```diff
@@ -314,7 +314,7 @@ def create_settings(self):
                 doc=LOAD_AS_CHOICE_HELP_TEXT)
 
         self.volumetric = cps.Binary(
-            text="Volumetric",
+            text="Data is 3D",
```

Comment:
@AnneCarpenter IIRC, this is why I went with “Volumetric.” Could you make a final decision? I’d like to merge.